### PR TITLE
ddl: clone the struct correctly

### PR DESCRIPTION
**Pull request by xhebox**
### What problem does this PR solve?

Issue Number: close None

Problem Summary: as title.

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: pkg/parser/digester.go [hunk 1]
```diff
@@ -145,6 +145,9 @@ type sqlDigester struct {
 	lexer  *Scanner
 	hasher hash2.Hash
 	tokens tokenDeque
+	// when mark==true, content will not be replaced by '?'
+	// but it we wrapped by unicode `<>`
+	marker bool
```
Comment (tangenta) [inline pkg/parser/digester.go:150]: What is this?

Comment (xhebox) [reply pkg/parser/digester.go]: Pulled changes from another branch. Deleted.

**Review by zimulala — COMMENTED** [review]
Could we add a test?

**Comment by zimulala** [comment]
@xhebox why close this PR?

**Comment by xhebox** [comment]
> @xhebox why close this PR?

I think it is an misoperation. I was cleaning useless old branches. When you delete a branch, the related opened PR will be closed.

**Pull request closed without merging**